https://github.com/PyFilesystem/pyfilesystem2/pull/588.patch

From 5c4bde407300dc7b4af6019c19219eaad59d93ee Mon Sep 17 00:00:00 2001
From: Eli Schwartz <eschwartz93@gmail.com>
Date: Sun, 15 Sep 2024 19:35:28 -0400
Subject: [PATCH] tests: remove test that doesn't seem to make a lot of sense

It claims to be testing that a `__repr__()` "works", but this doesn't
make any sense as everything eventually gets one from `object()`. It
also makes use of the deprecated pkg_resources interface, so simply
ditch this code entirely.
---
 tests/test_opener.py | 6 ------
 1 file changed, 6 deletions(-)

diff --git tests/test_opener.py tests/test_opener.py
index 43d56903..55f8c878 100644
--- tests/test_opener.py
+++ tests/test_opener.py
@@ -215,12 +215,6 @@ def setUp(self):
     def tearDown(self):
         shutil.rmtree(self.tmpdir)
 
-    def test_repr(self):
-        # Check __repr__ works
-        for entry_point in pkg_resources.iter_entry_points("fs.opener"):
-            _opener = entry_point.load()
-            repr(_opener())
-
     def test_open_osfs(self):
         fs = opener.open_fs("osfs://.")
         self.assertIsInstance(fs, OSFS)
